addon/components/mdc-grid-list/tiles/tile/primary/primary-content.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import layout from '../../../../../templates/components/mdc-grid-list/tiles/tile/primary/primary-content';
+
+export default Ember.Component.extend({
+  //region Ember Hooks
+  layout,
+  tagName: 'img',
+  classNames: ['mdc-grid-tile__primary-content'],
+  attributeBindings: ['src']
```

Comment:
We should be sure to also bind:

- `crossorigin`
- `height`
- `ismap`
- `longdesc`
- `referrerpolicy`
- `sizes`
- `srcset`
- `width`
- `usemap`

Reply:
Done!